MyApplication3/app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -0,0 +1,165 @@
+package com.example.myapplication;
+
+public class MatchData {
+
+    //Test Data
+    boolean testBool = false;
+
+    //Auto Data
+    boolean workingAuto = false;
+    boolean moveOutOfZone = false;
+    int ampAuto = 0;
+    int speakerAuto = 0;
+    enum ContactRobot {
+        None,
+        Allies,
+        Opposition
+    }
+    ContactRobot contactBot;
+
+    //Teleop Data
+    boolean broke = false;
+    boolean defense = false;
+    int ampTeleop = 0;
+    int ampSpeaker = 0;
+    int speakerTeleop = 0;
+    int dump = 0;
+    enum NoteAcquisition {
+        None,
+        Floor,
+        Source,
+        Both
+    }
+    NoteAcquisition noteAcquired;
+
+    //End of Match data
+    enum Chain {
+        Nope,
+        Attempted,
+        Succeeded
+    }
+    Chain chaining;
+
+    enum Trap {
+        Nope,
+        Attempted,
+        Succeeded
+    }
+    Trap trapping;
+    enum Harmony {
+        Nope,
+        Attempted,
+        Succeeded
+    }
+    Harmony harmo;
+    String notes = "";
+    // add human player spotlight data later
+
+
+    //Getters and Setters
+
+    //Test
+
+    public boolean getTestBool() {    return testBool;    }
+    public void setTestBool(boolean testBool) {    this.testBool = testBool;    }
+
+    //Auto
+
+    public boolean isWorkingAuto() {
+        return workingAuto;
+    }
+    public void setWorkingAuto(boolean workingAuto) {    this.workingAuto = workingAuto;    }

```

Comment:
This setter is implemented in a single line; the getter is in multiple lines, and other setters are in multiple lines as well. We should be consistent.